app/controllers/admin/volunteers.py

```diff
@@ -109,3 +110,14 @@ def updateBackgroundCheck():
         type = eventData['bgType']
         setUserBackgroundCheck(user,type, checkPassed)
         return " "
+
+@admin_bp.route('/updateProgramManager', methods=["POST"])
+def updateProgramManager():
+    if g.current_user.isCeltsAdmin:
```

Comment:
There should be an `else` clause that throws a 403 if they aren't an admin, `abort(403)`